content/blog/2018-04-26-rectangling-onboarding.md

```diff
@@ -0,0 +1,497 @@
+---
+slug: rectangling-onboarding
+title: Rectangling onboarding
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+date: 2018-04-26
+categories: blog
+topicid: 925
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- ghql
+- jqr
+- git2r
+output: 
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+*This blog post is the first of a 3-post series about a data-driven
+overview of rOpenSci onboarding. Read the intro to the series
+[here](https://ropensci.org/blog/2018/04/26/satrday-ct-serie/).*
+
+Our [onboarding
+reviews](https://ropensci.org/blog/2017/09/11/software-review-update/)
+take place in the open, in the issue tracker of a GitHub repository.
```

Comment:
good to put the question(s) you're trying to answer right up front in the post so people know why to keep reading

Reply:
Not sure whether this is a request for change?